src/App/store/AppState.ts

```diff
@@ -226,7 +226,7 @@ export class Project implements BKProject {
 class ProjectList {
   constructor() {
     makeObservable(this);
-    ipcRenderer.on('did-finish-getbkproject', (_event: Event, projects: Project[]) => {
+    window.api.onBKProject((projects: Project[]) => {
```

Comment:
`projects` should also be singular here also (I know it was already like that but good too fix since we noticed).